gnmawsgr/tasks.py

```diff
@@ -276,12 +276,7 @@ def do_glacier_restore(request_id,itemid,path):
     while True:
         try:
             with open(filename,'wb') as fp:
-                try:
-                    key.get_file(fp, cb=partial(download_callback, rq), num_cb=40)
-                except AttributeError:
-                    raven_client.captureException()
-                    logger.error("No Key found in S3. Object values: {0}".format(rq.__dict__))
-                    return
```

Comment:
what's the reason for removing the explicit capture of the exception here? was this causing a problem? I would be worried if this could fail silently.